storage/storage_setup.py

```diff
@@ -19,57 +19,37 @@ def schema_setup(ip_addresses, username, password, keyspace):
 
     #Create the Knowledge Base tables and related Materialized Views
     session.execute(
-        """CREATE TABLE IF NOT EXISTS object_knowledge_base_table(
+        """CREATE TABLE IF NOT EXISTS knowledge_base_table(
         object_id text,
         feature_type text,
-        feature_value text,
+        feature_value blob,
         timestamp timeuuid,
-        PRIMARY KEY ((object_id), feature_type, feature_value, timestamp)
+        PRIMARY KEY ((object_id), feature_type, timestamp)
         );
         """
         )
 
-    session.execute(
-        """CREATE MATERIALIZED VIEW IF NOT EXISTS mv_feature_value_table AS
-        SELECT * FROM object_knowledge_base_table
-        WHERE object_id IS NOT NULL
-        AND feature_type IS NOT NULL
-        AND feature_value IS NOT NULL
-        AND timestamp IS NOT NULL
-        PRIMARY KEY ((feature_value), feature_type, object_id, timestamp);
-        """
-        )
-
-    session.execute(
-        """CREATE MATERIALIZED VIEW IF NOT EXISTS mv_feature_type_table AS
+    session.execute("""CREATE MATERIALIZED VIEW IF NOT EXISTS mv_feature_type_table AS
```

Comment:
I would recommend the same for this table. Maybe name it mv_analytics_relationship_features_by_feature_type